examples/hello_rp.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/sh
```

Comment:
I think the name of the example is too generic and confusing as we have already `helloworld_mpi.py` and `helloworld_threads.py`. Maybe `helloworld_cu_startup.py`? Don't like it much but I did not want to complain without leaving a positive suggestion :)

Reply:
hehe.  Well, the script probably should go toward integration testing really, it is not very illustrative of a user workload, and thus not a good example per se.  OTOH, it does a thorough job on inspecting task setup and placement, and can be used to confirm scheduler placement (which is what I mainly use it for) - but I did not have a better place to put it.  `tests/scripts/` maybe?

Reply:
I was just thinking about naming but yes, `tests/scripts/` seems good. @lee212 maybe a useful starting point for some of your integration tests?